examples/thread_example.rb

```diff
@@ -0,0 +1,39 @@
+require 'securerandom'
+require 'gilmour/backends/redis'
+
+def redis_send_and_recv(message, key, ident)
+  redis = Gilmour::RedisBackend.new({})
+  redis.setup_subscribers({})
+  #loop do
+    waiter = Thread.new { loop { sleep 1 } }
+    newkey = "#{key}.#{SecureRandom.hex(2)}"
```

Comment:
A couple of puts still left. Also, if you want to have debug statements, use `$stderr.puts` to direct them to stderr. Ideally we should move them all to a logging framework.